packages/e2e-tests/test-applications/nextjs-app-dir/app/route-handlers/[param]/error/route.ts

```diff
@@ -1,3 +1,3 @@
-export async function PUT() {
+export async function PUT(): Promise<Response> {
```

Comment:
For some reason the new version wants types here. I think nextjs has some kind of ts plugin that checks for this.